core/primitives/Cargo.toml

```
@@ -35,6 +35,8 @@ near-rpc-error-macro = { path = "../../tools/rpctypegen/macro" }
 [features]
 default = ["jemallocator"]
 dump_errors_schema = ["near-rpc-error-macro/dump_errors_schema"]
+nightly_protocol = ["forward_chunk_parts"]
+forward_chunk_parts = []
```

To be able to run nightly protocol with only a selected set of features we should be able to decouple the complete list of features from the nightly mode itself. As I proposed in https://github.com/nearprotocol/NEPs/issues/126#issuecomment-712130812, let's use `nightly_protocol` as a marker feature and `nightly_protocol_features` listing all the nightly features we are ready to test in the nightly. Thus, you can test EVM exclusively (no other nightly protocol will get involved):

```
$ cargo run --features nightly_protocol --features protocol_feature_evm
```

```suggestion
nightly_protocol = []
nightly_protocol_features = ["nightly_protocol", "protocol_feature_forward_chunk_parts"]
protocol_feature_forward_chunk_parts = []
```